entities/ether.go

```diff
@@ -1,15 +1,23 @@
 //go:generate go run github.com/tinylib/msgp -unexported -tests=false -v
-//msgp:tuple Ether
+//msgp:tuple etherInner
+//msgp:ignore Ether
 
 package entities
 
+import "github.com/tinylib/msgp/msgp"
+
+// etherInner is the main usage of a 'native' currency, i.e. for Ethereum mainnet and all testnets
+type etherInner struct {
+	*BaseCurrency
+}
```

Comment:
not sure if directly defining `type etherInner BaseCurrency` would save any memory, maybe we can explore this if we really want to optimize - it's optional